app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::UsersController < ApplicationController
+        def create
+          user = User.new(user_params)
+          if user.save
+            render json: { data: {user_id: user.id,name: user.name} }, status: :created
+          else
```

Comment:
I think we might want a session to be created when a new user creates an account too so they don't have to log in after they create the account in the application. Not the worst thing in the world to not implement, but would be a nice touch I think.

Also, I think we should probably utilize a serializer for the user as well. That way it can be used across the user and session controller and just send back the all the info that's needed with less customization/configuration in different places should we change anything down the line.